### DISCUSSION TITLE
Error when trying to install cx_Freeze on Windows

### DISCUSSION THREAD

**Discussion by pedrogabrielLima** [marcelotduarte/cx_Freeze#1394] [Q&A] [answered]
Hello, I'm trying to install cx_Freeze through pip install and it's give me an error: 

- "**error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/**".

I tried many solutions, like following the instalation options in: 
   - https://cx-freeze.readthedocs.io/en/latest/installation.html

### Also, I followed the steps returned by the error console message, and installed the Visual Studio C++ 16.11.10, which is greater than the minimum required(14.0), and I'm still having the error when trying to install, and of course, I restarted my PC after installation of VS C++. _Please, help-me, I don't know what to do anymore to solve this problem._

**Here is a printscreen from powershell:**

- ![error_cx_Freeze](https://user-images.githubusercontent.com/22032685/156078371-2a78fdc8-98e5-4550-b062-02fb273865ec.png)

**Here is a printscreen from VS C++ manager(_showing that I'm up to date with de tool_):**

- ![VisualStudioCPP](https://user-images.githubusercontent.com/22032685/156078376-136205b2-bf77-47c5-93e6-ad2240b98c1e.png)

**Comment by marcelotduarte** [comment, marked as answer]
Please check this [comment](https://github.com/marcelotduarte/cx_Freeze/issues/1384#issuecomment-1041507835)

**Reply by pedrogabrielLima** [reply]
Thankyou, it solved the problem, I followed theses steps:

1.  pip install -f https://lief-project.github.io/packages/lief/ lief>=0.12.0.dev0

2. pip install -i https://marcelotduarte.github.io/packages/ cx_Logging

3. pip install cx_Freeze

If you speak portuguese(_your name looks like a brazilian name_), **aqui vai meu muitíssimo obrigado!**

**Reply by marcelotduarte** [reply]
Falou! ;-)
